modules/markdown.php

```diff
@@ -6,7 +6,7 @@
  * Sort Order: 31
  * First Introduced: 2.8
  * Requires Connection: No
- * Auto Activate: No
+ * Auto Activate: Yes
```

Comment:
This has the potential to change the front-end content of their site.  Are we okay with that?